src/components/Banner/index.js

```diff
@@ -31,11 +35,6 @@ function Banner({ mainTitle, items }) {
 }
 
 Banner.propTypes = {
-  items: PropTypes.array,
-  mainTitle: PropTypes.string,
```

Comment:
These should be kept. In the `propTypes` property you define the type of your props regardless if they are required or not. The only difference is that if they aren't required, then you need to add a default value in the `defaultProps` property.